src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -382,13 +382,18 @@ private function getProvider(PropertyInterface $property)
      */
     private function getCurrentPage($pageParameter)
     {
-        if ($this->requestStack->getCurrentRequest() !== null) {
-            $page = $this->requestStack->getCurrentRequest()->get($pageParameter, 1);
-        } else {
-            $page = 1;
+        if ($this->requestStack->getCurrentRequest() === null) {
+            return 1;
+        }
+
+        $page = $this->requestStack->getCurrentRequest()->get($pageParameter, 1);
+        if ($page < 1) {
```

Comment:
I think a 404 error would be the correct solution. Else you display "wrong" content on a "invalid" url and this could lead to duplicate content.